airbyte-integrations/connector-templates/source-python-cdk/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM airbyte/integration-base-python:0.1.5
+
+# Bash is installed for more convenient debugging.
+RUN apt-get update && apt-get install -y bash && rm -rf /var/lib/apt/lists/*
+
+ENV CODE_PATH="source_{{snakeCase name}}"
+ENV AIRBYTE_IMPL_MODULE="source_{{snakeCase name}}"
+ENV AIRBYTE_IMPL_PATH="Source{{properCase name}}"
+
+WORKDIR /airbyte/integration_code
+COPY $CODE_PATH ./$CODE_PATH
+COPY setup.py ./
+RUN pip install .
+
+LABEL io.airbyte.version=0.1.1
```

Comment:
is there any reason this isn't `0.1`? I see `0.1.1` everywhere. no complaints - just curious to understand,

Reply:
this should be 0.1.0